app/package.json

```diff
@@ -27,6 +27,7 @@
     "@types/lodash": "^4.14.168",
     "babel-polyfill": "6.26.0",
     "bip39-light": "^1.0.7",
+    "bitbox02-api": "^0.11.0",
```

Comment:
When building the app, I'm getting

```
WARNING in ./app/node_modules/bitbox02-api/lib/bitbox02-api-go.js 4:181-188
Critical dependency: require function is used in a way in which dependencies cannot be statically extracted
 @ ./app/node_modules/bitbox02-api/lib/bitbox02.js 9:21-52
 @ ./app/node_modules/bitbox02-api/lib/index.js 45:14-38
 @ ./app/frontend/wallet/shelley/shelley-bitbox02-crypto-provider.ts 298:45-106
 @ ./app/frontend/wallet/shelley/shelley-crypto-provider-factory.ts 3:0-79 11:23-52
 @ ./app/frontend/actions/wallet.ts 49:0-93 179:45-91
 @ ./app/frontend/actions.ts 23:0-45 26:440-453
 @ ./app/frontend/components/autoLogout.ts 16:0-33 78:7-14
 @ ./app/frontend/components/app.tsx 10:0-38 35:38-48
 @ ./app/frontend/walletApp.js 96:14-49
```

I recall that this might have been discussed, but is this going to get fixed? Perhaps in the new version that benma suggested?

Reply:
yes, I called out the same issue in the original PR, there doesn't seem to be a simple fix to it right now, but should be solved in some later release of the library: https://github.com/vacuumlabs/adalite/pull/1187#issuecomment-963408318